plugins/ruby-foreman-discovery/debian/control

```diff
@@ -8,6 +8,6 @@ Homepage: https://github.com/theforeman/foreman_discovery
 
 Package: ruby-foreman-discovery
 Architecture: all
-Depends: ${misc:Depends}, bundler, foreman (>= 1.24.0~rc1)
+Depends: ${misc:Depends}, bundler, foreman-assets(>= 2.0.0~rc1), foreman (>= 1.24.0~rc1)
```

Comment:
assets needs to be in BuildDepends -- line 5

and the versions of assets and foreman should match ;)

Reply:
ah! my bad.